dev_install.md

```diff
@@ -15,7 +15,7 @@ Steps:
 
 ### Configuration
 
-Create a `.env` file at root of the project containing the following keys:
+Create a `resources/.env` file containing the following keys:
```

Comment:
Since we're always passing this file as a flag anyways, can we just call it icubam.env?  Or is this a default path/

Reply:
This is the default value in some scripts (populate_db_fake, migrate_db, populate_db, synchronize_db), but  not for run_server. Argh...
Do you want me to set default value to `resources/icubam.env` for every body ? (+ run_server, schedule_sms, send_sms)

Reply:
If you feel like it's not too much probably good to standardize, and therefore might as well make the file visible.